example/Mock.elm

```diff
@@ -1,45 +1,94 @@
 module Mock exposing (..)
 
-{-| imports for examples
-    >>> import Dict exposing (Dict, fromList)
-    >>> import String exposing (join)
+{-|
+
+@docs add, sum, bar, rev, IsItCool, coolFunc
+
+    -- Imports for doc-tests
```

Comment:
This would be more obvious as English (and not code) if it said "These imports are only used in doc-tests"